perfkitbenchmarker/benchmark_spec.py

```diff
@@ -191,11 +191,24 @@ def Prepare(self):
   def Delete(self):
     if FLAGS.run_stage not in ['all', 'cleanup'] or self.deleted:
       return
+
     if self.vms:
-      vm_util.RunThreaded(self.DeleteVm, self.vms)
-    self.firewall.DisallowAllPorts()
+      try:
+        vm_util.RunThreaded(self.DeleteVm, self.vms)
+      except Exception as e:
+        logging.error('Got an exception: %s\n'
```

Comment:
Consider `logging.exception` for these as well so the stack trace is printed, unless you don't think it will be informative.